test/router-luatest/router_test.lua

```diff
@@ -444,3 +444,72 @@ g.test_ssl = function(g)
     vtest.router_cfg(g.router, cluster_cfg)
     vtest.storage_wait_fullsync(g)
 end
+
+g.test_enable_disable = function(g)
+    --
+    -- gh-291: router enable/disable
+    --
+    local router_name = 'router_1'
+    local router = vtest.router_fiber_new(g, router_name, cluster_cfg,
```

Comment:
If you follow my advice above about not having a dedicated `router_fiber_new`, then you also won't need this `pre_func` arg and can call it right in the test. Without bringing it up to vtest.